medusa/providers/torrent/html/bjshare.py

```diff
@@ -0,0 +1,257 @@
+                        if serie in title:
+                            # remove season from title when its in absolute format
+                            title = re.sub('S\d{2}(E\d{2,4})', r'\1', title)
+                            break
+
+                    download_url = urljoin(self.url, result.select('a[href^="torrents.php?action=download"]')[0]['href'])
+                    if not all([title, download_url]):
+                        continue
+
+                    seeders = try_int(cells[labels.index('Seeders')+group_index].get_text(strip=True))
+                    leechers = try_int(cells[labels.index('Leechers')+group_index].get_text(strip=True))
+
+                    # Filter unseeded torrent
+                    if seeders < self.minseed or leechers < self.minleech:
+                        if mode != 'RSS':
+                            log.debug("Discarding torrent because it doesn't meet the"
+                                      " minimum seeders: {0}. Seeders: {1}",
+                                      title, seeders)
+                        continue
+
+                    torrent_details = u''
+                    if "group_torrent" in result_class:
+                        # torrents belonging to a group
+                        torrent_details = title
+                        title = group_title
+                    elif "torrent" in result_class:
+                        # standalone/un grouped torrents
+                        torrent_details = cells[labels.index('Nome/Ano')].find('div', class_='torrent_info').get_text()
+
+                    torrent_details = torrent_details.replace('[', ' ').replace(']', ' ').replace('/', ' ')
+                    torrent_details = torrent_details.replace('Full HD ', '1080p').replace('HD ', '720p')
+
+                    torrent_size = cells[labels.index('Tamanho')+group_index].get_text(strip=True)
+                    size = convert_size(torrent_size) or -1
+
+                    torrent_name = '{0} {1}'.format(title, torrent_details.strip()).strip()
+                    torrent_name = re.sub('\s+', ' ', torrent_name)
+
+                    items.append({
+                        'title': torrent_name,
+                        'link': download_url,
+                        'size': size,
+                        'seeders': seeders,
+                        'leechers': leechers,
+                    })
+
+                    if mode != 'RSS':
+                        log.debug('Found result: {0} with {1} seeders and {2} leechers'.format
+                                  (torrent_name, seeders, leechers))
+
+                except StandardError:
+                    continue
+        return {'has_next_page': has_next_page, 'items': items}
+
+    def login(self):
+        """Login method used for logging in before doing a search and torrent downloads."""
+        return self.cookie_login('Login', check_url=self.urls['search'])
```

Comment:
Use default login methods if there is no captcha.

Reply:
It does require captcha... it's just not always but something trigger it from time to time, i'm not sure what exacly, i had problem with that already, that's why i did with cookies...